search/localities_source.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <cstdint>
+
+namespace search
+{
+struct LocalitiesSource
+{
+  LocalitiesSource();
+
+  template <typename Fn>
+  void ForEachType(Fn && fn) const
+  {
+    fn(m_city);
+    fn(m_town);
+  }
+
+  uint32_t m_city = 0;
+  uint32_t m_town = 0;
```

Comment:
It's optional. Don't you want to write the constructor this way:
```c++
struct LocalitiesSource
{
  LocalitiesSource::LocalitiesSource() : m_city(classif().GetTypeByPath({"place", "city"})), 
      m_town(classif().GetTypeByPath({"place", "town"}))
   {
   }
...
  uint32_t m_city;
  uint32_t m_town;
};
```
?

Reply:
No, thank you, but I'd prefer to keep this code as is.

Reply:
Ok